packages/host/app/components/search-sheet/index.gts

```diff
@@ -157,23 +158,23 @@ export default class SearchSheet extends Component<Signature> {
   getCard = restartableTask(async (cardURL: string) => {
     this.clearSearchCardResults();
 
-    let response = await this.loaderService.loader.fetch(cardURL, {
-      headers: {
-        Accept: 'application/vnd.card+json',
-      },
-    });
-    if (response.ok) {
-      let maybeCardDoc = await response.json();
-      if (isSingleCardDocument(maybeCardDoc)) {
-        let card = await this.cardService.createFromSerialized(
-          maybeCardDoc.data,
-          maybeCardDoc,
-          new URL(maybeCardDoc.data.id),
-        );
+    let maybeCardDoc: CardDocument | undefined;
+    try {
+      maybeCardDoc = await this.cardService.fetchJSON(cardURL);
```

Comment:
same comment here--we should probably use  `getCard()` from the card resource instead of fetching the json and deserializing by hand